client-server/src/main/java/Client.java

```diff
@@ -0,0 +1,78 @@
+import java.io.*;
+import java.net.Socket;
+import java.net.SocketException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+public class Client {
+
+    private static int clientId;
+
+//  private static MessageStrategy messages = new SerializableStrategy();
+    private static MessageStrategy messages = new JsonStrategy();
+
+    private static String formatMessage(Message msg) {
+
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(Message.TIME_FORMAT_PATTERN);
+        StringBuilder sb = new StringBuilder();
+        sb.append("user#");
+        sb.append(msg.getUserId());
+        sb.append(" ");
+        sb.append(msg.getDateTime().format(formatter));
+        sb.append(": ");
+        sb.append(msg.getMessage());
+        return sb.toString();
```

Comment:
Probably it could be simplified if you use `String.format` instead:
`return String.format("user#%d %s: %s", msg.getUserId(), msg.getDateTime().format(formatter), msg.getMessage());`

Reply:
yes, it looks more elegant